core/util/google.py

```diff
@@ -20,7 +20,6 @@
 #from core.util import proxy
 class main:
 	# framework = None
-	countip=0
 	def __init__(self, q, limit=1, count=10, google_api=None, google_cx=None)#, autoproxy=False):
```

Comment:
Please always test your code before every push.
Missing the colon at the end.
After fixing that I got a ton of indentation errors.

Reply:
I am really sorry for the missing colon.

But for the code indentation, I am not finding errors yet. Can you please share some screenshots here @k4u5h1k ? It'll be a great help.

Reply:
If it works fine push it. Could've been an error on my side.